sdk/synapse/synapse-spark/src/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 1.0.0-beta.2 (Unreleased)
+
+## 1.0.0-beta.1 (2020-11-10)
```

Comment:
we don't put the date until it is released, and you haven't released yet, right?

Reply:
Yes, will remove it at next commit.